wire-ios-data-model/Source/ManagedObjectContext/CoreDataStack.swift

```diff
@@ -185,6 +186,16 @@ public class CoreDataStack: NSObject, ContextProvider {
     }
 
     deinit {
+        close()
+    }
+
+    public func close() {
```

Comment:
As an uninitiated reader I would expect that `close()` sub sequentially also calls `closeStores()`?

Reply:
we do call it